bootstrap/bootstrap.go

```diff
@@ -99,11 +99,28 @@ func startIngestion(
 ) error {
 	logger.Info().Msg("starting up event ingestion")
 
-	client, err := grpc.NewClient(cfg.AccessNodeGRPCHost)
+	grpcClient, err := grpc.NewClient(cfg.AccessNodeHost)
+	if err != nil {
+		return fmt.Errorf("failed to create client connection for host: %s, with error: %w", cfg.AccessNodeHost, err)
+	}
+
+	client, err := models.NewCrossSporkClient(grpcClient, logger)
```

Comment:
I believe we could encapsulate the instantiation logic for `CrossSporkClient`, with a method that accepts `cfg.AccessNodeHost` & `cfg.AccessNodePreviousSporkHosts` and does the necessary creation of clients.
Maybe not something addressable in this PR, but a note to consider.